Cargo.toml

```diff
@@ -61,7 +61,7 @@ serde_yaml = "0.9.19"
 
 [lib]
 path="src/lib.rs"
-name = "zenohc"
+name = "zenohcd"
```

Comment:
The name of the library wants to change again...

Reply:
and other issues